srcpkgs/nheko/template

```diff
@@ -1,18 +1,26 @@
 # Template file for 'nheko'
 pkgname=nheko
-version=0.7.2
-revision=3
+version=0.8.2
+revision=1
 build_style=cmake
 hostmakedepends="qt5-host-tools qt5-qmake pkg-config qt5-declarative"
 makedepends="qt5-multimedia-devel qt5-svg-devel qt5-tools-devel fmt-devel
- tweeny spdlog mtxclient-devel boost-devel cmark-devel olm-devel json-c++
- libsodium-devel lmdb++ qt5-declarative-devel qt5-quickcontrols2-devel"
+ spdlog mtxclient-devel boost-devel cmark-devel olm-devel json-c++ lmdb++
+ qt5-declarative-devel qt5-quickcontrols2-devel qtkeychain-qt5-devel
+ gst-plugins-bad1-devel gst-plugins-base1-devel"
 depends="hicolor-icon-theme qt5-quickcontrols2 qt5-graphicaleffects
- qt5-multimedia"
+ qt5-multimedia gst-plugins-good1 gst-plugins-good1-qt5"
 short_desc="Desktop client for Matrix using Qt and C++14"
 maintainer="Lorem <notloremipsum@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/Nheko-Reborn/nheko"
 changelog="https://github.com/Nheko-Reborn/nheko/blob/master/CHANGELOG.md"
 distfiles="https://github.com/Nheko-Reborn/nheko/archive/v${version}.tar.gz"
-checksum=412908fcec0e66a0557db5359556b65e85dd22b8644a6a2ed569a90dea4bde3c
+checksum=df4575c47daab47d418637a4637b599e3848dd749132ca1e92f981d9212eabb2
+
+case "$XBPS_TARGET_MACHINE" in
+	armv*-musl)
+		makedepends+=" libunwind-devel"
+		configure_args+=" -DCMAKE_CXX_STANDARD_LIBRARIES=-lunwind" ;;
+	*) ;;
+esac
```

Comment:
Can you test a local build if `armv6l` doesn't need `libunwind` either? And possibly test if `armv7l-musl` actually requires it.